packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -126,7 +126,7 @@
 +  fluentd:
 +    repository: rancher/mirrored-kube-logging-fluentd
 +    tag: v1.16-4.8-full
```

Comment:
Does this tag apply to this version of logging?

In this case the 2.9 chart is still targeting 4.8.0 of upstream charts. Whereas the 2.10 charts is tracking 4.10.0, so we may want to take care when backporting to avoid any changes there that are specific to 2.10's chart.